### PR TITLE
login page added

### PR DESCRIPTION
**Pull request by iamknk**

### CODE REVIEW

File: app/layout.tsx [hunk 1]
```diff
@@ -1,23 +1,33 @@
...
 import { Inter } from "next/font/google";
 import "./globals.css";
 import { cn } from "../lib/utils";
 import SideNavbar from "@/components/SideNavbar";
+import { useEffect, useState } from "react";
+import { isLoginPage } from "@/lib/utils";
 
 const inter = Inter({ subsets: ["latin"] });
 
-export const metadata: Metadata = {
```
Comment (SvenLigensa) [inline app/layout.tsx:11]: Why did you remove that? Please add again

**Review by SvenLigensa — CHANGES_REQUESTED** [review]
If I load the page, on the first load, the side navigation bar does not show. Could you fix that? Otherwise good.

File: app/layout.tsx (hunk 1 above)
Comment (iamknk) [reply app/layout.tsx:11]: It was removed to fix issue of adding custom sidebar that should not be visible on login page. So layout was made a client side component.

We can add meta data using <head> tag within the layout.
